parsl/launchers/launchers.py

```diff
@@ -81,11 +81,10 @@ def __call__(self, command, tasks_per_node, nodes_per_block):
         - task_block (string) : bash evaluated string.
 
         """
-        task_blocks = tasks_per_node * nodes_per_block
 
         x = '''export CORES=$(getconf _NPROCESSORS_ONLN)
 echo "Found cores : $CORES"
-WORKERCOUNT={3}
+WORKERCOUNT={nodes_per_block}
```

Comment:
If I understand correctly, this changes the original values from `total workers` to `total nodes`, correct?

Reply:
Oh, this is an off-by-one error in my patch. I was starting counting at 1.